ourpackage/barcelos.py

```diff
@@ -0,0 +1,119 @@
+import os
+from colorama import Fore 
+
+def kevin():
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("4. Comment from Veslino")
+        print("5. Comment from Consultado")
+        print("6. Comment from Andaya")
+        print("0. To go back")
```

Comment:
I want to add my comment to your code - Piadozo

Reply:
You have done it.